docs/changelog.txt

```diff
@@ -1,3 +1,54 @@
+0.5.0 (2020-04-26)
+------------------------------------------------------------------------
+
+- Feature: Industries now support up to 16 accepted cargos and 16 produced cargos
+- Feature: Houses now support up to 16 accepted cargos
+- Feature: Support for RoadTypes (a.k.a NRT)
+- Feature: Allow use of switches and random switches as procedures, reducing repetition of switches
+- Feature: Increase railtype ID range from 16 to 64, to support 64 railtypes
+- Feature: Railtype flags RAILTYPE_FLAG_ALLOW_90DEG and RAILTYPE_FLAG_DISALLOW_90DEG for 90 degree curves
+- Feature: Railtype flags RAILTYPE_FLAG_HIDDEN and RAILTYPE_FLAG_PRECOMBINED for precombined spriteset type
+- Feature: Add ANIM_TRIGGER_APT_AIRPLANE_LANDS (follows OpenTTD #7182)
+- Feature: Add station var nearby_tile_grfid (0x6A)
+- Feature: PUSH/POP_COLOUR codes for strings
+- Feature: Support industry vars production_rate (0x70) and transported_last_month_pct (0x71)
```

Comment:
Reorder so this is with the 16-cargo industry change above? IIRC this was needed to support production rates for all those cargoes.